docs/examples/use_cases/detection_pipeline.ipynb

```diff
@@ -200,7 +200,7 @@
     "labels = labels_cpu.at(img_index)\n",
     "categories_set = set()\n",
     "for label in labels:\n",
-    "    categories_set.add(label[0])\n",
+    "    categories_set.add(label)\n",
```

Comment:
only code where labels were expected to have an extra dim. In the detection pipelines the labels go to the box encoder, and the extra dimension is flattened there.